onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java

```diff
@@ -259,44 +266,62 @@ private void changePreferenceSummary(String preferenceKey) {
         if (preferenceKey.equalsIgnoreCase(getString(R.string.preference_key_region))
                 || preferenceKey.equalsIgnoreCase(getString(R.string.preference_key_oba_api_url))) {
             if (Application.get().getCurrentRegion() != null) {
-                regionPref.setSummary(Application.get().getCurrentRegion().getName());
-                customApiUrlPref
+                mPreference.setSummary(Application.get().getCurrentRegion().getName());
+                mCustomApiUrlPref
                         .setSummary(getString(R.string.preferences_oba_api_servername_summary));
+                String customOtpApiUrl = Application.get().getCustomOtpApiUrl();
+                if (customOtpApiUrl == null) {
```

Comment:
Looking at the code below line 289, I think this should be `if (!TextUtils.isEmpty(customOtpApiUrl))`?